docs/documentation/getting-started/deployment/manual/stack.md

```diff
@@ -112,12 +112,53 @@ clusters using the kind tool as described in the
 
         before you proceed.
 
+<!-- markdownlint-disable MD013 -->
+1. Configure TLS to enable secure communication to, and between the VCPs:
```

Comment:
It may be interpreted as this step could be skipped if TLS ist not desired (like in case of development). However this would break the next step (launching VCPs) as it requires the environment variable `PROTOCOL` to be defined to work properly